documentation/index.html

```diff
@@ -324,6 +324,40 @@ <h6 class="heading-6">This is h6 heading</h6>
         <iframe src="https://www.thiscodeworks.com/embed/6202473ded723a001551c17c" style="width: 100%; height: 294px;" frameborder="0"></iframe>
       </div>
       <!-- Text utils end -->
+      <!-- List Component -->
+      <div id="list" class="ml-1 mb-1 newline">
+        <h3>List</h3>
+        <p class="text-content">Spaced Lists</p>    
+        <div class="p-1 flex">
+          <ul class="spaced-list box-shadow ml-1 p-1 pl-2">
+            <li class="list-normal px-1">Item 1</li>
+            <li class="list-normal px-1">Item 2</li>
+            <li class="list-normal px-1">Item 3</li>
+          </ul>
+          <ul class="spaced-list box-shadow ml-1 p-1">
+            <li class="text-decoration-none px-1">Item 1</li>
```

Comment:
what is px-1 , can give better name here

Reply:
left right padding, padding on x axis